### PR TITLE
final-fantasy.ch - Fix for input (sending) button

### PR DESCRIPTION
**Pull request by Oreloth**
Adjusting colors for input button: 
Before: image
After: image

### CODE REVIEW

**Comment by Gusted** [comment]
Hi @Oreloth!

Where can I find this element?

**Comment by Oreloth** [comment]
Hi @Gusted!
All pages with an input area, I assume you need an account to see comments button, I didn't find page without an account needed, sorry for that.

**Comment by Gusted** [comment]
Alright, I'm a bit suspicious of why these fixes are needed are these kind of elements should in practice not require any fixes.

**Comment by Oreloth** [comment]
I understand your doubt, anyway I suggest this fix for users who will want to set a comment, as you can see in my screenshots, the "Envoyer (Send)" button is totally unreadable.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -7362,7 +7362,12 @@ final-fantasy.ch
 CSS
 nav ul li:first-child span,
 h1 {
-    color: #000 !important;
+color: #000 !important;
+}
+input[type="submit" i],
+input[type="file" i] {
+background-color:#444;
+text-color:white;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7370]:
```suggestion
    color: #000 !important;
}
input[type="submit" i],
input[type="file" i] {
    background-color: ${#c7c7c7} !important;
    color: var(--darkreader-neutral-text) !important;
```

Comment (Oreloth) [reply src/config/dynamic-theme-fixes.config:7370]: Testing...

**Comment by Oreloth** [comment]
Result:
image

**Review by Gusted — APPROVED** [review]

**Pull request merged**